### PR TITLE
[aavegotchi-agip-17] new strategy for agip17

### PR DESCRIPTION
**Pull request by candoizo**
Changes proposed in this pull request:
- updates aavegotchi-agip strategy to calculate realm voting power

### CODE REVIEW

File: src/strategies/aavegotchi-agip/index.ts [hunk 1]
```diff
@@ -85,9 +88,26 @@ export async function strategy(
...
-    const { id, gotchisOwned } = addrInfo;
+  result.users.map(({ id, gotchisOwned, parcelsOwned }) => {
+    let realmVotingPowerValue = 0;
+    if (parcelsOwned.length > 0) {
+      parcelsOwned.map(({ size }) => {
+        let votePower = realmSizeVotePower[size];
+        if (isNaN(votePower)) votePower = 0;
+        realmVotingPowerValue += votePower;
+      });
+    }
```
Comment (ChaituVR) [inline src/strategies/aavegotchi-agip/index.ts:109]: Hi @candoizo Existing proposals could be affected by this change, especially the proposal that is open 
https://snapshot.org/#/aavegotchi.eth/proposal/0x396844d504394f7f335f070d443c33455300ee21d90db6c4b089760a3e0469bf

It is better to create a `-v2` strategy for this

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/aavegotchi-agip/index.ts (hunk 1 above)
Comment (candoizo) [reply src/strategies/aavegotchi-agip/index.ts]: alright ive made the changes requested

File: src/strategies/aavegotchi-agip-17/index.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  137: 'https://api.thegraph.com/subgraphs/name/aavegotchi/aavegotchi-core-matic'
+};
+
+export async function strategy(
+  _space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
```
Comment (ChaituVR) [inline src/strategies/aavegotchi-agip-17/index.ts:16]:
```
'snapshot' is defined but never used
```
Hi @candoizo is there some way to pass snapshot block number into the subgraph? without it, the voting power may change in the future or in the middle of voting a proposal etc. the VP should be based on snapshot block number to have consistent results

File: src/strategies/aavegotchi-agip-17/examples.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "name": "AGIP 17: Voting power for GHST value of parcels",
+    "strategy": {
+      "name": "aavegotchi-agip-17",
+      "params": {
+        "tokenAddress": "0x86935f11c86623dec8a25696e1c19a8659cbf95d",
```
Comment (ChaituVR) [inline src/strategies/aavegotchi-agip-17/examples.json:7]: This is not used anymore?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/aavegotchi-agip-17/index.ts (hunk 2 above)
Comment (candoizo) [reply src/strategies/aavegotchi-agip-17/index.ts:16]: good point, fixed in recent commit thanks

**Review by bonustrack — APPROVED** [review]

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**